src/Microsoft.Extensions.Options.ConfigurationExtensions/ConfigureFromConfigurationOptions.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.Extensions.Options
 {
-    public class ConfigureFromConfigurationOptions<TOptions> : ConfigureOptions<TOptions>
+    public class ConfigureFromConfigurationOptions<TOptions> : ConfigureOptions<TOptions>, IOptionsConfiguration<TOptions>
```

Comment:
Lou had proposed Alteration, I suggested your OptionsConfiguration since I remembered you liked that previously.  I always liked Setup personally…